pkgs/development/tools/build-managers/bazel/cpp-test.nix

```diff
@@ -2,7 +2,8 @@
   bazel
 , bazelTest
 , bazel-examples
-, gccStdenv
+, stdenv
+, darwin
```

Comment:
I agree with @layus's [comment](https://github.com/NixOS/nixpkgs/pull/186106#issuecomment-1231287176) that this is confusing. If the strange double meaning (`darwin = stdenv` on Linux) can't be avoided, then a comment explaining it may be helpful.

Reply:
As this is in the tests, and not in the main bazel_6 derivation, I think we can ignore this for now. Unless it changes the test significantly ?